specifications/sep-payment-channel-mechanism.md

```diff
@@ -194,7 +194,7 @@ multisig accounts. F has source account E, and sequence number set to s.
     - One or more `CHANGE_TRUST` operations configuring trustlines on EI.
     - One `SET_OPTIONS` operation adjusting escrow account EI's thresholds such
     that I and R's signers must both sign.
-    - One or more `SET_OPTIONS` operations adding I's signers to ER.
+    - One or more `SET_OPTIONS` operations adding I and R as equal signers to ER.
```

Comment:
Subtle distinction here that we want to add I and R's signers to each escrow account, not necessarily I and R specifically. I and R may have multi-sig accounts. The code doesn't do that yet, nor do we need to prioritize the code doing that yet, but the SEP can speak into that future.
```suggestion
    - One or more `SET_OPTIONS` operations adding I and R's signers to ER.
```

Reply:
I left the word `equal` in there as well. I feel like that's important to point out since ensuring their equal (and the other party didn't secretly boost their weight of the escrow account) is one of the reason to add both signers and not just 1
https://github.com/stellar/experimental-payment-channels/pull/31/commits/d30bcd9972d949b649059822162df31f42c6ec62

Reply:
I need to think about the equal part some more. They may not be equal weight, even though the weights allow for equal control. The line further up is the important bit:
>adjusting escrow account EI's thresholds such that I and R's signers must both sign.

Reply:
hermm ok. I guess I assumed they'd always be equal, but I guess doesn't have to be. I'll remove
https://github.com/stellar/experimental-payment-channels/pull/31/commits/9a8eb1f380ea4614e4471895c73d237eadfba8c5